src/ucp/core/ucp_ep.inl

```diff
@@ -156,42 +156,49 @@ static UCS_F_ALWAYS_INLINE ucp_ep_flush_state_t* ucp_ep_flush_state(ucp_ep_h ep)
     return &ucp_ep_ext_gen(ep)->flush_state;
 }
 
-static UCS_F_ALWAYS_INLINE uintptr_t ucp_ep_dest_ep_ptr(ucp_ep_h ep)
+static UCS_F_ALWAYS_INLINE ucs_ptr_map_key_t ucp_ep_rkey(ucp_ep_h ep)
```

Comment:
using `...rkey` name creates confusion with well established memory keys
maybe use `pkey`  (peer key), or make some other distinction with memory keys

Reply:
I agree and discussed this question with @evgeny-leksikov - it's key because of the hash context (key & value).
I also think we should have a more suited name. maybe ep_map_id?
(pkey already has another meaning in IB)

Reply:
every EP contains local and remote keys, we can use ID naming like lid/rid, but LID is also used in IB. `ep_map_id` (then probably should be `ep_worker_map_id`) is too long and creates dependency on implementation.

Reply:
how about local_ep_id and remote_ep_id then?